src/test/java/com/hocs/test/glue/DataInputStepDefs.java

```diff
@@ -58,12 +64,9 @@ public void completeDataInputPerCaseType(String caseType) {
 
     @When("^I add an additional correspondent$")
     public void iAddAnAdditionalCorrespondent() {
-        clickOn(recordCorrespondentDetails.additionalCorrespondentYesRadioButton);
-        clickOn(dataInput.continueButton);
-        clickOn(dataInput.correspondentMemberNoRadioButton);
-        clickOn(continueButton);
-        recordCorrespondentDetails.fillMandatoryCorrespondentFields();
-        clickOn(continueButton);
+        addACorrespondentThatIsOrIsNotAnMP("Is not");
+        recordCorrespondentDetails.fillMandatoryCorrespondentFieldsForSecondaryContact();
+        dataInput.clickAddButton();
```

Comment:
Try to stick to `clickOn();`

Reply:
amended underlying clickAddButton() method to use clickOn() instead of .click()